sky/backends/monkey_patches/monkey_patch_ray_up.py

```diff
@@ -15,7 +15,9 @@
 Reasons:
  - With our patch, the ray up will always reuse the existing head node with the same cluster
```

Comment:
https://github.com/skypilot-org/skypilot/pull/1671#discussion_r1098121032

Btw, can we document how & where worker nodes are handled?